TombEngine/Objects/Generic/Traps/dart_emitter.cpp

```diff
@@ -76,65 +79,21 @@ namespace TEN::Entities::Traps
 			dartItem->ObjectNumber = ID_DARTS;
 			dartItem->RoomNumber = item->RoomNumber;
 
-			int x = 0;
-			int z = 0;
-
-			switch (item->Pose.Orientation.y)
-			{
-			case 0:
-				z = WALL_SIZE / 2;
-				break;
-
-			case 0x4000:
-				x = WALL_SIZE / 2;
-				break;
-
-			case -0x8000:
-				z = -WALL_SIZE / 2;
-				break;
-
-			case -0x4000:
-				x = -WALL_SIZE / 2;
-				break;
-			}
-
-			dartItem->Pose.Position.x = x + item->Pose.Position.x;
-			dartItem->Pose.Position.y = item->Pose.Position.y - WALL_SIZE / 2;
-			dartItem->Pose.Position.z = z + item->Pose.Position.z;
+			dartItem->Pose.Position.x = item->Pose.Position.x ;
+			dartItem->Pose.Position.y = item->Pose.Position.y -CLICK(0.9);
+			dartItem->Pose.Position.z = item->Pose.Position.z;
 
 			InitialiseItem(dartItemNumber);
 
-			dartItem->Pose.Orientation.x = 0;
-			dartItem->Pose.Orientation.y = item->Pose.Orientation.y + -ANGLE(180);
+			dartItem->Pose.Orientation.x = item->Pose.Orientation.x + -ANGLE(180);
```

Comment:
can this be simplified to just `item->Pose.Orientation.y - ANGLE(180)`?

Reply:
fixed